biomage/stage/stage.py

```diff
@@ -162,34 +160,7 @@ def get_branch_ref(chart, token, repo_to_ref=None, return_sha=False):
     raise Exception("Invalid repository supplied.")
 
 
-def get_all_experiments(source_table="experiments-staging"):
```

Comment:
Removing this as we do not fetch from DynamoDB anymore. We use the return value (experiment names) to check against sandboxId (see line 252) which doesn't make sense.